config.py

```diff
@@ -115,7 +115,8 @@ class Config:
 
     FRONTEND_ADDRESS = 'https://dm-dev.apps.staging.digital.gov.au'
     ADMIN_ADDRESS = 'https://dm-dev-admin.apps.staging.digital.gov.au'
-    REACT_APP_ROOT = '/2'
+    MARKETPLACE_APP_ROOT = '/2'
```

Comment:
make our life easier - make the marketplace app root match the framework slug i.e. `dm`